src/pytest_bdd/parser.py

```diff
@@ -337,25 +339,27 @@ class Examples:
 
     line_number: int | None = field(default=None)
     name: str | None = field(default=None)
+    tags: set[str] = field(default_factory=set)
```

Comment:
this might not be need anymore now that I know the flow

Reply:
If you don't have it, you need to have
```suggestion
    tags: set[str] | None = field(default=None)
```
and then you have to deal with the possibility of it being `None` everywhere you deal with it (which is harder than it should be, because `pylint` doesn't know how to narrow types within conditional logic).

So I think it's better to keep this as-is, and then you always know that `tags` is a `set`.

Reply:
I'll keep it as a set for now, since I think thats what they also do in the other two spots the initialize the other tags

Reply:
I think `pytest-bdd` it is designed so that tags can either be an empty set or an initialized set. I don't know if any of the places that use `field(...)` have any logic to check for the `None` type. I might dig into this more to try and solidify my understanding.